release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3116,6 +3116,68 @@ $ ccoctl azure create-all \
 --
 (link:https://issues.redhat.com/browse/OCPBUGS-22651[*OCPBUGS-22651*])
 
+* When installing an {product-title} cluster with static IP addressing and Tang encryption, worker nodes boot without network settings. This condition prevents worker nodes from accessing the Tang server, causing installation to fail. To address this condition, you must set the network settings for each worker node as `ip` kernel arguments.
++
+When using installer-provisioned installation, provide `ip` kernel arguments for each worker node using a manifest. For example: 
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 99-worker-tang-network-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+  kernelArguments:
+    - 'ip=<static-ip>::<gateway>:<netmask>:<hostname-1>:<interface>:none'
+    - 'ip=<static-ip>::<gateway>:<netmask>:<hostname-2>:<interface>:none'
+    - 'rd.neednet=1'
+----
++
+Save and copy the manifest file to the `clusterconfigs/openshift` directory:
++
+[source,terminal]
+----
+$ cp ~/router-replicas.yaml clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+Apply the manifest: 
++
+[source,terminal]
+----
+$ oc apply -f clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+When installing with the {ai-full}, modify the infrastructure environment using the API to append the network settings as `ip` kernel arguments for each worker node. For example:
++
+[source,bash]
```

Comment:
Per OpenShift documentation guidelines, use `[source,bash]` only for [bash scripts](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output:~:text=Note-,For%20bash%20scripts,-%2C%20use%20%5Bsource%2Cbash).
```suggestion
[source,terminal]
```